src/java.xml/share/classes/module-info.java

```diff
@@ -27,11 +27,70 @@
  * Defines the Java API for XML Processing (JAXP), the Streaming API for XML (StAX),
  * the Simple API for XML (SAX), and the W3C Document Object Model (DOM) API.
  *
+ * <h2 id="ConfigurationFile">JAXP Configuration File</h2>
+ * JAXP supports the use of a configuration file for the
+ * <a href="#LookupMechanism">Factory Lookup Mechanism</a> and
+ * setting properties that have defined corresponding system properties.
+ *
+ * <h3>Format</h3>
+ * The configuration file must be in standard {@link java.util.Properties} format.
+ * <p>
+ * The keys are the names of the system properties, for example, those listed in
+ * column {@code System Property Name} of the table <a href="#Factories">JAXP Factories</a>,
+ * or {@code System Property} in the table {@code Catalog Features}
+ * of class {@link javax.xml.catalog.CatalogFeatures CatalogFeatures}.
+ *
+ * <p>
+ * The values are those defined in the specific API or process, for example, the
+ * fully qualified name of the implementation class for the
+ * <a href="#LookupMechanism">Factory Lookup Mechanism</a>, or {@code value} in
+ * table {@code Catalog Features} of class
+ * {@link javax.xml.catalog.CatalogFeatures CatalogFeatures}.
+ *
+ * <p>
+ * Below are examples on what can be placed in the configuration file:
+ * <pre>
+ *     {@code javax.xml.parsers.DocumentBuilderFactory=packagename.DocumentBuilderFactoryImpl}
+ *     {@code javax.xml.catalog.resolve=strict}
+ * </pre>
+ *
+ *
+ * <h3 id="CF_Default">{@code jaxp.properties}</h3>
+ * By default, the <a href="#Factories">JAXP Factories</a> will look for a
+ * configuration file called {@code jaxp.properties} in the {@code conf} directory
+ * of the Java installation and use the entries if any to customize the behavior
+ * of the factories.
+ *
+ * <p>
+ * {@code jaxp.properties} will be read only once during the initialization of
+ * the JAXP implementation and cached in memory. If the file does not exist when
+ * the first attempt is made to read from it, no further attempts are made to check
+ * for its existence. It is not possible to change the value of any property after
+ * it has been read for the first time.
+ *
+ *
+ * <h3 id="CF_SP">User-defined Configuration File</h3>
+ * A user-defined configuration file can be set outside of the JDK by using the
+ * system property {@code java.xml.config.file}.
```

Comment:
"can be set out of the JDK" needs to be re-phrased. Maybe this sentence can be restructured to say that a system property can be set on the command line to specify the location of a configuration file on the file system. Adding "command line" would be helpful because "system property" is too overloaded in these docs.

Reply:
Updated. Note it can also be set through the System API.